images/runtime/Dockerfile

```diff
@@ -35,6 +35,7 @@ COPY iptables-wrapper /usr/sbin/iptables-wrapper
 COPY configure-iptables-wrapper.sh /tmp/configure-iptables-wrapper.sh
 RUN /tmp/configure-iptables-wrapper.sh
 
+# XXX: perhaps these should be copied into /usr/local/bin, so that it's easy to copy-out
```

Comment:
leftover? I think copied into `/usr/local/bin` makes sense if it's not too much work to do it so.

Reply:
At this stage it would require a rebuild of LLVM image, hence I've not fixed this right away.

Reply:
Yeah, I think so, also in each of the upstream images.